multi-arch-builders/coreos-s390x-rhcos-builder.bu

```diff
@@ -91,3 +102,54 @@ storage:
 
             echo "Importing tarball into volume"
             sudo -u builder -H /bin/bash -c "cd /var/home/builder; podman volume import secex-data /var/home/builder/${TARBALL}"
+
+    - path: /usr/local/bin/create-cex-config.sh
+      mode: 0755
+      user:
+        name: root
+      group:
+        name: root
+      contents:
+        inline: |
+            #!/bin/bash
+            set -e
+
```

Comment:
Can we add some reference URLs here to official IBM docs for the steps done here?

Reply:
Will do that